snull/snull.c

```diff
@@ -90,7 +90,8 @@ struct snull_priv {
 	struct napi_struct napi;
 };
 
-static void snull_tx_timeout(struct net_device *dev);
+//static void snull_tx_timeout(struct net_device *dev);
```

Comment:
@jypan-ccu this logic also fails on 5.4 kernel builds.  I've addressed with https://github.com/martinezjavier/ldd3/commit/4a9147796b94f0659eadbda57e2bfb2eb6246da8

Reply:
Perfect! It works, except the git program is complicated for me.